src/main/java/com/google/devtools/build/lib/remote/Retrier.java

```diff
@@ -136,6 +137,43 @@ public void recordFailure() {}
         public void recordSuccess() {}
       };
 
+  public static class FailureRateCircuitBreaker implements CircuitBreaker {
+
+    private static final int MIN_NUMBER_OF_EXECUTIONS = 5;
```

Comment:
I feel like we can use a longer variable name to make it clearer what is the variable for, e.g. `MIN_EXECUTIONS_TO_COMPUTE_FAILURE_RATE`